doc/user-guide/embed-sql-py.md

```diff
@@ -0,0 +1,73 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst:
+  html_meta:
+    description lang=en: Embed SQL queries in .py file
+    keywords: jupyter, sql, jupysql
+    property=og:locale: en_US
+---
+
+# Cell magics in .py scripts
+
+Several users may want to run SQL queries in a Python script instead of Jupyter Notebook. In this tutorial, we'll demonstrate how to embed JupySQL magics in a Python file using VSCode.
```

Comment:
Suggestion: I would start with an introduction to what JupySQL does, then explain how users can also use it within VSCode. For example:

"We have seen JupySQL's capabilities to easily run SQL queries within Jupyter notebooks, but what if you want to execute SQL queries in a .py script instead?".

Reply:
Made the change